lighthouse-core/report/v2/report-generator.js

```diff
@@ -79,9 +80,11 @@ class ReportGeneratorV2 {
   generateReportHtml(reportAsJson) {
     const sanitizedJson = JSON.stringify(reportAsJson).replace(/</g, '\\u003c');
     const sanitizedJavascript = REPORT_JAVASCRIPT.replace(/<\//g, '\\u003c/');
+    const sanitizedCss = REPORT_CSS.replace(/<\//g, '\\u003c/');
```

Comment:
We'd know right away if the whole report was broken :) The current `_escapeScriptTags` does the same as your `sanitizedJson`, which sanitizes user input in the json results. But our own stuff should be safe (`REPORT_JAVASCRIPT` and `REPORT_CSS`)